### DISCUSSION TITLE
Won't Build On Fedora 34

### DISCUSSION THREAD

**Discussion by SteveGremory** [hecrj/iced#863] [Q&A]
So i've tried everything I could, maybe this is an issue with my path variables, but i don't think so. What's happening is that when I build iced, for some reason it fails when it's building the dependancy, "servo-fontconfig-sys v5.1.0", even though I have all of _it's_ dependancies installed. [Here's the log.](https://pastebin.com/DLaxLHCN) Here's what I think is the, ["important part of the log."](https://pastebin.com/ENnaxNVM)
Thanks!

**Comment by 13r0ck** [comment]
This help?
```
sudo dnf install libXcursor-devel-0:1.2.0-2.fc32.x86_64
sudo dnf install libX11-devel-0:1.6.12-1.fc32.x86_64
sudo dnf install libXrandr-devel-0:1.5.2-3.fc32.x86_64
```
Install newer versions if there are any

**Comment by SteveGremory** [comment]
Nope, Sorry but that didn't work.

**Comment by FabianLars** [comment]
Are you sure you have the freetype dev package installed? On Ubuntu it is `libfreetype6-dev`, so it should be something like `freetype` or `freetype-devel` on fedora.
At least that was the problem i had many many times, with the same error in the log...

**Comment by SteveGremory** [comment]
> Are you sure you have the freetype dev package installed? On Ubuntu it is `libfreetype6-dev`, so it should be something like `freetype` or `freetype-devel` on fedora.
> At least that was the problem i had many many times, with the same error in the log...

Nope, that's all already installed. That's the problem.

**Comment by 13r0ck** [comment]
Have you re-installed the deps?

**Comment by SteveGremory** [comment]
> Have you re-installed the deps?

Yes, I have. It ofc just said, "this is already installed"

**Comment by CicadaCinema** [comment]
On Fedora 33 I was able to fix this error by installing `freetype-devel`.
